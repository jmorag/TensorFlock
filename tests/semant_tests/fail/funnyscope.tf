main = f True;

f : Bool -> Nat;
f b = b + m; {
  m : Nat;
  m = if b then 42 else 24;
  b : Nat;
  b = 12;
}
