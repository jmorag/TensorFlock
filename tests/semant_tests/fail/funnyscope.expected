Fatal error: exception Failure("Non-boolean expression in if statement")
